### PR TITLE
Improve defines in 'iotjs_binding.h' to generate better code.

### PR DESCRIPTION
**Pull request by LaszloLango**
IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
**RPi2 `.text` sizes**

|  | .text (default) | .text (target) |
| - | ---- | ---- |
|master| 294961 | 372004 |
| branch | 294833 | 371820 |

File: src/iotjs_binding.h [hunk 1]
```diff
@@ -108,6 +108,11 @@ jerry_value_t iotjs_jhelper_eval(const char* name, size_t name_len,
                                  const uint8_t* data, size_t size,
                                  bool strict_mode);
 
+/* Note:
+ *      Defines started with underscores should not be used
+ *      outside of this header.
```
Comment (hs0225) [inline src/iotjs_binding.h]: How about using `#undef`?

Comment (LaszloLango) [reply src/iotjs_binding.h]: Good idea. I'd look better with `#undef`. I'll update the patch.

Comment (hs0225) [reply src/iotjs_binding.h]: Sorry. This case cannot do because it is a recursive definition.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**